codespell_lib/data/dictionary.txt

```diff
@@ -13311,6 +13373,7 @@ sesssions->sessions
 setgit->setgid
 seting->setting
 setings->settings
+setis->set is
```

Comment:
Perhaps suggest settees although that may be too far.

Reply:
Done